pkg/lint/rules/template.go

```diff
@@ -50,7 +50,7 @@ func Templates(linter *support.Linter, values map[string]interface{}, namespace
 	path := "templates/"
 	templatesPath := filepath.Join(linter.ChartDir, path)
 
-	templatesDirExist := linter.RunLinterRule(support.WarningSev, path, validateTemplatesDir(templatesPath))
+	templatesDirExist := linter.RunLinterRule(support.ErrorSev, path, validateTemplatesDir(templatesPath))
 
 	// Templates directory is optional for now
 	if !templatesDirExist {
```

Comment:
This check should be removed instead. This then would then remove most if not all updates to the unit tests. It might be worth though adding a unit test for an error in a subchart if it does not exist already.

Reply:
> This check should be removed instead. This then would then remove most if not all updates to the unit tests. It might be worth though adding a unit test for an error in a subchart if it does not exist already.

Do you mean that you don't verify the `template`, just remove it?